Modules/_testcapi/func_events.c

```diff
@@ -1,11 +1,16 @@
 #include "parts.h"
 
+#define Py_BUILD_CORE
+#include "pycore_function.h"  // FUNC_MAX_WATCHERS
+
 #define NUM_WATCHERS 2
 static PyObject *pyfunc_watchers[NUM_WATCHERS];
 static int watcher_ids[NUM_WATCHERS] = {-1, -1};
 
-static PyObject *get_id(PyObject *obj) {
-    PyObject *builtins = PyEval_GetBuiltins();
+static
+PyObject *get_id(PyObject *obj)
```

Comment:
PEP 7 nit

```suggestion
static PyObject *
get_id(PyObject *obj)
```

Reply:
Do you use an automatic formatter like `clang-format`? If so, would you mind sharing your config?

Reply:
Sorry, I don't use an automatic formatter for CPython; I've got PEP 7 in my fingers :) I know there's been discussions about having a PEP 7 check in the CI or doing some kind of automatic formatting, but those ideas have been rejected every time.

See [this Discourse topic](https://discuss.python.org/t/pep-proposal-automatically-formatting-the-cpython-code/5603) for one of the most recent discussions.

Reply:
Paul Ganssle has a `clang-format` config that approximate PEP 7: https://gist.github.com/pganssle/0e3a5f828b4d07d79447f6ced8e7e4db